README.md

```diff
@@ -1,6 +1,64 @@
-# edgex-endorsement
+# EdgeX Endorsement
 
 ## Introduction
 
-Code, tools, and documentation for validating and endorsing device profiles written by third parties.
+Code, tools, and documentation for validating and endorsing device profiles written by third parties. At a high level you'll need to complete five steps to be eligible for the EdgeX Endorsement Program which will ensure that data provided meets all required components. 
+
+1) Deploy EdgeX using one of the provided cloud templates. Alternatively, you can leverage the latest released compose-file provided here: This ensures a vanilla version of EdgeX is being used without modification to ensure the highest compatibility.
+2)  Compose and deploy your device profile using the Device Profile Modeling tool provided [here](https://www.edgexfoundry.org/tbd). This will ensure the profile is validated and has all required components.
+2) Run your device locally, configured to point at the cloud instance of the deployed EdgeX to send data to either an MQTT or REST device service.
+3) Once deployed, Run the verification script provided to ensure that core-data has been correctly populated with the data that was sent in the previous step.
+4) Submit your device profile along with sample data to EdgeX for approval [here](https://www.edgexfoundry.org/tbd).
```

Comment:
Do we want the device profile?  Or are the words "device profile" placeholders for some yet to be defined evidentiary data proving a specific test was run.

If we accept device profiles, are we doing anything with them?

There might be some resistance to providing device profiles as they may only be made available when appropriate licensing agreements (e.g. money has been exchanged) are in place.

Reply:
to me the device profile submissions is a link to the github repo they are hosted in -- which in turn, once verified -- will land on the edgex endorsement webpage and be linked to.